src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-checkbox(
+  $thickness-base: 1px,
+  $thickness-max: 3px,
+  $border-color: $ecl-color-grey-25,
+  $border-color-focus: $ecl-color-yellow-100,
+  $border-color-checked: $ecl-color-blue-100,
+  $fake-checkbox-size: 0.875rem
+) {
+  .ecl-checkbox {
+    align-items: center;
+    display: flex;
+    position: relative;
+  }
+
+  .ecl-checkbox__input {
+    cursor: pointer;
+    margin-left: 3px;
+    opacity: 0;
+    padding: 0;
+    position: absolute;
+    z-index: 1;
+  }
+
+  .ecl-checkbox__label {
+    color: $ecl-color-grey-100;
+    font-size: $ecl-font-m;
```

Comment:
`$ecl-font-m` = `normal normal 400 1rem/1.25rem Arial,sans-serif;`
it's not what you want to use here ;)

Reply:
Can i ask you then what is the correct interpretation of a requirement defined as such? Font-size: M  :)